helm/prometheus-rules/templates/alerting-rules/fluentbit.rules.yml

```diff
@@ -28,6 +29,24 @@ spec:
         severity: notify
         team: atlas
         topic: observability
+    - alert: FluentbitDropRatio
+      annotations:
+        description: '{{`Fluentbit ({{ $labels.instance }}) is dropping more than 1% records.`}}'
+        opsrecipe: fluentbit-too-many-erros/
+      # Check the ratio of dropped records over the total number of records.
+      expr: rate(fluentbit_output_dropped_records_total[10m]) / (rate(fluentbit_output_proc_records_total[10m]) + rate(fluentbit_output_dropped_records_total[10m])) > 0.01
+      for: 10m
```

Comment:
Sure for only 10m

Reply:
Same as the previous `FluentbitTooManyErrors` alert.
This way, they should both pop at the same time, and correlation is easier.

Reply:
Same as the previous `FluentbitTooManyErrors` alert.
This way, they should both pop at the same time, and correlation is easier.